src/unittest/test_settings.cpp

```diff
@@ -72,7 +72,7 @@ const char *TestSettings::config_text_before =
 	"some multiline text\n"
 	"     with leading whitespace!\n"
 	"\"\"\"\n"
-	"np_terrain = 5, 40, (250, 250, 250), 12341, 5, 0.7, 2.4\n"
+	"np_terrain = 5, 40, (250, 250, 250), 12341, 5, 0.75, 2.25\n"
```

Comment:
would this actually fail with the old code?
I think we rather have to test `123.4567` or such